recipes/trouve/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "trouve" %}
+{% set version = "0.5.1" %}
+{% set author = "rwhitt2049" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
Please install from a tarball, not from the repo:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

You can see the canonical way of doing this in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml